pkgs/top-level/all-packages.nix

```diff
@@ -130,6 +130,8 @@ with pkgs;
 
   httpTwoLevelsUpdater = callPackage ../common-updater/http-two-levels-updater.nix { };
 
+  httpUpdater = callPackage ../common-updater/http-updater.nix { };
```

Comment:
Maybe it could be called `directoryListingUpdater` :woman_shrugging:

Reply:
Ok.